test/bindings/python/valhalla.json

```diff
@@ -1,14 +1,46 @@
 {
+  "mjolnir": {
+    "max_cache_size": 1000000000,
+    "id_table_size": 1300000000,
+    "use_lru_mem_cache": false,
+    "lru_mem_cache_hard_control": false,
+    "use_simple_mem_cache": false,
```

Comment:
oops.. seems like that's the current of `valhalla_build_config`.. or at least what the python `json` library makes of it. hope that's ok.